### PR TITLE
ref(grouping): Remove uniq_id experiment

### PR DESCRIPTION
**Pull request by mikejihbe**
This experiment is still too broad to be safely generally applicable. I no longer believe we can avoid tokenization of message strings.

### CODE REVIEW

File: src/sentry/grouping/strategies/message.py [hunk 1]
```diff
@@ -52,13 +52,13 @@
         )
     ) |
     (?P<uuid>
-        \b
+        (\b|_)
```
Comment (armenzg) [inline src/sentry/grouping/strategies/message.py:55]: Are you reverting to its original value?

File: tests/sentry/grouping/test_normalize_message.py [hunk 2]
```diff
@@ -119,25 +119,30 @@ def record_analytics(monkeypatch):
             """blah connection failed after 12345ms 1.899s 3s""",
             """blah connection failed after <duration> <duration> <duration>""",
         ),
+        # (
```
Comment (armenzg) [inline tests/sentry/grouping/test_normalize_message.py:122]: If we're not going to use it, should we be deleting these lines instead?

File: tests/sentry/grouping/test_normalize_message.py [hunk 3]
```diff
@@ -119,25 +119,30 @@ def record_analytics(monkeypatch):
...
-            """fbtrace_id Aba64NMEPMmBwi_cPLaGeeK AugPfq0jxGbto4u3kxn8u6p blah""",
-            """fbtrace_id <uniq_id> <uniq_id> blah""",
-        ),
-        (
-            "Uniq ID - word with numerical pre/suffix",  # TODO: It is possible to have fbtrace_ids without integers.
-            """1password python3 abc123 123abc""",
-            """1password python3 abc123 123abc""",
+            "UUID after underscore",
+            "[words] look come-look_18d34d42-1aaa-6bac-bce3-4c4a854061d2: true",
+            "[words] look come-look<uuid>: true",  # TODO: Should we parameterize bools like this?
```
Comment (armenzg) [inline tests/sentry/grouping/test_normalize_message.py:145]: Is this reverting to the original values? It's hard to understand from the diff if we're just reverting or not.

**Comment by mikejihbe** [comment]
closing in favor of https://github.com/getsentry/sentry/pull/67415

**Pull request closed without merging**